tox.ini

```diff
@@ -78,6 +78,8 @@ deps =
     coverage[toml]
     responses==0.20.0
     -r{toxinidir}/requirements.txt
+    # OpenTelemetry is in PYDEPS for tempo instrumentation
+    opentelemetry-exporter-otlp-proto-grpc
```

Comment:
in traefik, I added an optional_dependencies field to pyproject.toml called `pydeps`.
then in tox, in all envs that need the charm libs, you can write `.[pydeps]` and it will load them from pyproject.toml. 
I think it's nicer to have a single place where we need to keep them up to date.

But it's just a nice to have.